en/docs/reference/ballerina-module/overview.md

```diff
@@ -0,0 +1,94 @@
+The WSO2 Micro Integrator module for Ballerina enables integration developers to leverage Ballerina's powerful transformation capabilities within WSO2 MI. Ballerina, being a cloud-native programming language with built-in support for JSON and XML, simplifies data transformations compared to traditional Class Mediators. The module also provides access to Ballerina's extensive ecosystem of language modules and connectors, enhancing connectivity with external systems.
+
+## Prerequisites
+
+- WSO2 Micro Integrator 4.2.0 or later
+- Ballerina 2201.12.0 or later
+- Java Development Kit (Version 17 for MI 4.2.0/4.3.0, Version 21 for MI 4.4.0)
+- VS Code with WSO2 Micro Integrator extension installed
+- Ballerina VS Code extension installed
+
+## Get Started with Ballerina Module for MI
+
+There are two ways to use the Ballerina module in your WSO2 MI projects:
+
+1. Using the WSO2 MI VS Code Extension
+2. Using the Command Line Interface (CLI)
+
+### Method 1: Use WSO2 MI VS Code Extension
+
+1. Install the [WSO2 Micro Integrator VSCode Extension](https://mi.docs.wso2.com/en/latest/develop/mi-for-vscode/install-wso2-mi-for-vscode/).
+2. [Create](https://mi.docs.wso2.com/en/latest/develop/create-integration-project/) a new integration project or open an existing project.
+3. Expand the artifact types in the `Add artifact` view and select **Ballerina Module**.
```

Comment:
```suggestion
3. Expand the artifact types in the **Add artifact** view and select **Ballerina Module**.
```

Reply:
Check other places as well. The style is is to use **bold text** to refer UI elements.